platipy/imaging/visualisation/visualiser.py

```diff
@@ -1028,146 +1026,150 @@ def _overlay_contours(self):
             "y": (0, size_sag, 0, size_ax),
             "z": (0, size_sag, 0, size_cor),
         }
+        with warnings.catch_warnings():
```

Comment:
This is the bit that suppresses the warning :)

Reply:
Looks good to me!